### PR TITLE
Add myrepos script

### PR DESCRIPTION
**Pull request by lgierth**
Ported over from ipfs/devtools#1 -- not porting over the prose for now.

### CODE REVIEW

**Comment by RichardLitt** [comment]
Can you point to `myrepos`?

File: dev/mkmrconfig.sh [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+more_repos=()
+
+more_repos+=("ipfs/ipfs,git@github.com:ipfs/ipfs.git")
+more_repos+=("ipfs/community,git@github.com:ipfs/community.git")
+more_repos+=("ipfs/specs,git@github.com:ipfs/specs.git")
+more_repos+=("ipfs/infrastructure,git@github.com:ipfs/infrastructure.git")
+more_repos+=("ipfs/ipfs-pages,git@github.com:ipfs/ipfs-pages.git")
+more_repos+=("ipfs/autobatch,git@github.com:whyrusleeping/autobatch.git")
+more_repos+=("ipfs/failstore,git@github.com:whyrusleeping/failstore.git")
+more_repos+=("ipfs/retry-datastore,git@github.com:whyrusleeping/retry-datastore.git")
```
Comment (RichardLitt) [inline dev/mkmrconfig.sh]: Why are these hardcoded?

File: dev/mkmrconfig.sh [hunk 2]
```diff
@@ -0,0 +1,82 @@
+#!/usr/bin/env bash
+set -eo pipefail; [[ $MKMRCONFIG_TRACE ]] && set -x
+
+# Scans Github organizations and emits a config file for use with `myrepos`.
+#
+# Dependencies:
+#   myrepos -- https://myrepos.branchable.com
```
Comment (RichardLitt) [inline dev/mkmrconfig.sh]: This looks like it isn't updated frequently. Can you find the github repo and point to it?

Comment (lgierth) [inline dev/mkmrconfig.sh]: It's the official project homepage and "Last edited 22 days and 4 hours ago" but yeah it's debatable whether that's the best entrypoint. Gonna add the github url and install url.

Comment (RichardLitt) [inline dev/mkmrconfig.sh]: Sounds good.

**Pull request merged**